ament_cmake_mypy/ament_cmake_mypy-extras.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Canonical, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Please restore the comment explicitly mentioned in the discussion with the changes suggested: https://github.com/ament/ament_lint/pull/154/commits/1ef5070e4c0aa917d9c6d67a058ca6f4e89194d2#diff-22d942d6a85569fa14706a02112cc9d1R15

Reply:
fixed in f088f5f